src/cleanvision/__init__.py

```diff
@@ -18,3 +19,5 @@ def get_version() -> str:
     __version__ = get_version()
 except Exception:
     pass
+
+Imagelab = _Imagelab
```

Comment:
just curious why this is necessary instead of just `from cleanvision.imagelab import Imagelab` ?

Reply:
It works, but flake8 throws an error for unused import. We should keep it in case someone accidentally removes the import statement through editor as it is unused.